config/initializers/high_voltage.rb

```diff
@@ -0,0 +1,3 @@
+HighVoltage.configure do |config|
+  config.home_page = 'home'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.